Src/IronPython/Runtime/TypecodeOps.cs

```diff
@@ -123,6 +123,8 @@ class TypecodeOps {
                     result = BitConverter.GetBytes(Convert.ToSingle(obj));
                     return true;
                 case "d":
+                    double objAsDouble = 0;
```

Comment:
What is this for? Doesn't seem to be used.

Reply:
Removed.